### ISSUE TITLE
Is it intentional that 24.10.1 and 24.10.2 don't provide manylinux2014 x86_64 wheels?

### ISSUE THREAD

**Issue by godlygeek** [gevent/gevent#2068]
gevent 24.10.1 and 24.10.2 have manylinux2014 compatible wheels on PyPI for some architectures (aarch64, ppc64le, and s390x) but not for x86_64; the x86_64 wheels support only manylinux_2_28 and up. Older versions did provide manylinux2014 compatible wheels for x86_64. Was this change intentional, or an oversight?

It was introduced by https://github.com/gevent/gevent/commit/74ef8762bbd5404f9423723c1e76d7496d14f2b8 but that commit doesn't explain why the change was made.

**Comment by jamadden** [comment]
Yes, it was deliberately dropped. manylinux2014 went EOL in June of this year.

**Comment by jamadden** [comment]
Why I missed updating the other architectures, that I can't tell you.

**Comment by godlygeek** [comment]
OK, understandable. manylinux2014 is EOL, yes, but still reasonably widely used. RHEL 7 uses manylinux2014, and is now in [Extended Lifecycle Support](https://www.redhat.com/en/blog/announcing-4-years-extended-life-cycle-support-els-red-hat-enterprise-linux-7), though admittedly out of security support.

All of which is to say, I can absolutely understand why you wouldn't want to ship manylinux2014 wheels, but it would make my life easier if you did, at least for as long as it's not a maintenance burden to do so.
